api/api_sources/resources/csv/SoilTextureCode.csv

```diff
@@ -3,3 +3,4 @@ C  ,Coarse (Sand / cobbles / gravel - water drains quickly)
 F  ,Fine (Clay - compact soil that holds water)
 M  ,Medium (Loamy / silty soil - water takes longer to drain)
 O  ,Organic (Dark soil with organic material)
+N  ,None

```

Comment:
Currently,  we are running the migration progressively, so make sure new values should be migrated as fresh new migration.